recipes/nb_black/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "nb_black" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33247d55b7bfa8d1d0725c7ba0dbd1984b386ef790d5bd1f7eb7c8826cdcfea5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
This needs to be made Python 3.6+ only or use selectors as it has different dependencies on 2.7: https://github.com/dnanhkhoa/nb_black/blob/master/setup.py#L28-L29

Reply:
As we explicitly depend on `black`, this recipe should already be Python 3.6+ but I can make this more visible in the `run` requirements.